index.html

```diff
@@ -3455,7 +3477,7 @@ <h3><dfn>Switch To Frame</dfn></h3>
    <dt><var>id</var> is <a><code>null</code></a>
    <dd>
     <ol>
-     <li><p>Set the <a>current browsing context</a> to
+     <li><p><a>Set the current browsing context</a> with
```

Comment:
Please note that the check for the `current browsing context` above is still faulty and wouldn't allow switching to the top-level browsing context at all if the current one is gone, but returns early.